vendor/github.com/docker/licensing/client.go

```diff
@@ -185,6 +187,10 @@ func (c *client) DownloadLicenseFromHub(ctx context.Context, authToken, subscrip
 
 func (c *client) ParseLicense(license []byte) (*model.IssuedLicense, error) {
 	parsedLicense := &model.IssuedLicense{}
+	// The file may contain a leading BOM, which will choke the
```

Comment:
Do we know where this BOM comes from? Have we found this in licenses we sent, or is this because a user manually edited the file?

Reply:
The old license server stuffed this header on there, and customers with old licenses lying around will have them.  The new license server doesn't do this so that's why we didn't notice the glitch until we started testing with some old offline licenses.

Reply:
Ah, that's unfortunate. I was hoping it was something we could fix on the server before release, but looks like that's not the case.